--- vim_old.vim    Wed Nov 20 07:20:47 2013
+++ vim_new.vim    Wed Nov 20 07:21:22 2013
@@ -103,7 +103,7 @@

 " All vimCommands are contained by vimIsCommands. {{{2
 syn match vimCmdSep    "[:|]\+"    skipwhite nextgroup=vimAddress,vimAutoCmd,vimCommand,vimExtCmd,vimFilter,vimLet,vimMap,vimMark,vimSet,vimSyntax,vimUserCmd
-syn match vimIsCommand    "\<\h\w*\>"    contains=vimCommand
+syn match vimIsCommand    "\<\h\w*\>"    contains=vimCommand,vimMap
 syn match vimVar        contained    "\<\h[a-zA-Z0-9#_]*\>"
 syn match vimVar        "\<[bwglsav]:\h[a-zA-Z0-9#_]*\>"
 syn match vimFBVar      contained   "\<[bwglsav]:\h[a-zA-Z0-9#_]*\>"
@@ -474,7 +474,7 @@

 " Additional IsCommand, here by reasons of precedence {{{2
 " ====================
-syn match    vimIsCommand    "<Bar>\s*\a\+"    transparent contains=vimCommand,vimNotation
+syn match    vimIsCommand    "<Bar>\s*\a\+"    transparent contains=vimCommand,vimMap,vimNotation

 " Highlighting {{{2
 " ============
